etc/kayobe/pulp.yml

```diff
@@ -134,69 +120,74 @@ stackhpc_pulp_publication_deb_development: "{{ stackhpc_pulp_distribution_deb_de
 # Content is made available at well-known distribution names for development,
 # production.
 
-# Development tracks the release repositories.
-stackhpc_pulp_distribution_deb_development:
-  # Base Ubuntu Focal repositories
-  - name: "ubuntu-focal-development"
-    repository: Ubuntu focal
-    base_path: "ubuntu/focal/development"
-    mode: verbatim
-    state: present
-    required: "{{ stackhpc_pulp_sync_ubuntu_focal | bool }}"
-  - name: "ubuntu-focal-security-development"
-    repository: Ubuntu focal security
-    base_path: "ubuntu/focal-security/development"
-    mode: verbatim
-    state: present
-    required: "{{ stackhpc_pulp_sync_ubuntu_focal | bool }}"
+stackhpc_pulp_repository_deb_repos: >-
+  {%- set deb_repos = [] -%}
+  {%- for repo in stackhpc_pulp_deb_repos -%}
+  {%- set deb_repo = {"name": repo.name} -%}
```

Comment:
Currently this ignores the `required` attribute. It gets used in e.g. `etc/kayobe/ansible/pulp-repo-sync.yml`.

Reply:
Bah, missed this - updated